tests/end-to-end/backup/chart/backup-test/values.yaml

```diff
@@ -7,4 +7,4 @@ image:
   dir:
   namePrefix: end-to-end-backup
   nameSuffix: tests
-  tag: PR-7800
+  tag: PR-7800
```

Comment:
Is the new line removal intended? I think the newline is good to have.

Reply:
currently there is no rule enforced, so as long as it is valid yaml id say it is ok